lib/work_stats.rb

```diff
@@ -74,6 +71,21 @@ def update_stat_counter
     )
   end
 
+  # Do not count the visit if:
+  # - This is the same visitor as before
+  # - The current user is a creator of this work
+  # - The request comes from a unicorn which sets the REQUEST_FROM_BOT enviroment variable
+  # (We will set this in the unicorn which serves bots, which is chosen by nginx.)
+  # - The work is hidden by an admin (admins can visit it but it shouldn't count as a hit)
+  # - The work is part of an unrevealed collection
+  private def disregard_visit?(visitor)
```

Comment:
I think we usually put `private` above the method rather than next to it. Is that possible here?

Reply:
Sure, the `protected` below will switch modes. (I got this habit from work, where we prefer it in order to not accidentally `private`ise a whole section of file. 😅 )

Reply:
However, indenting stuff under a modifier is somewhat unusual, and I'd rather not do it. 😬  (I'll ignore the indented `protected` section because it's already there, but I'd rather not repeat the pattern.)

Reply:
That's fine! I don't think we usually do it, tbh.

(I actually really like the explicitness of putting `private` on the same line as the method name, but probably best to stick with the style we already use.)